music21/musicxml/m21ToXml.py

```diff
@@ -1634,8 +1634,9 @@ def setMeterStream(self):
         #                meterStream, meterStream[0]])
         if not meterStream:
             # note: this will return a default if no meters are found
-            meterStream = s.flatten().getTimeSignatures(searchContext=False,
-                                                        sortByCreationTime=True, returnDefault=True)
+            meterStream = s.getTimeSignatures(searchContext=False,
+                                              sortByCreationTime=True,
+                                              returnDefault=True)
```

Comment:
Oh, spectacular.  yes, that would work.  We change "if meterStream is None" to "if not meterStream"

Reply:
yes!

Reply:
Actually, I'm going to do this as a separate PR, because from what it seems, neither meterStream nor refStreamOrTimeRange are needed here really.  I'm going to make a bigger PR to rename refStreamOrTimeRange to make it clearer.